src/api/CqmModelConversionService.ts

```diff
@@ -97,11 +108,147 @@ export class CqmConversionService {
 
     // TODO: need UI checkbox to determine yes/no
     cqmMeasure.calculate_sdes = false;
-    // TODO: build population_sets- MAT-5779
-    cqmMeasure.population_sets = null;
+    const populationSets: PopulationSet[] =
+      this.creatingPopulationSets(measure);
+    cqmMeasure.population_sets = populationSets;
     return cqmMeasure;
   }
 
+  private creatingPopulationSets = (measure: Measure) => {
```

Comment:
`buildCqmPopulationSets` would describe the action more accurately.

Reply:
Sure, I'll change it